styx-scheduler-service/src/main/java/com/spotify/styx/monitoring/Stats.java

```diff
@@ -57,5 +57,9 @@
 
   void exitCodeMismatch();
 
+  void registerResourceConfigured(String resource, Gauge<Long> resourceCount);
+
+  void resourceUsed(String resource, long used);
```

Comment:
Or, maybe I'm misunderstanding what's the difference between the `Gauge<>` ones and primitive ones.